### DISCUSSION TITLE
Do you think we will ever get the RSS timetags converted into RSS or MP3 chapters (pocketcasts recently added podlove & podcast index RSS chapters, it may be easier for you to do it now 🤔🤔)

### DISCUSSION THREAD

**Discussion by CookieyedCodes** [listenbox/community#17] [Q&A] [answered]
Or should I keep pushing pocketcasts to convert RSS timetags to chapters which would you think is easier?

**Comment by MtndaleRedneck** [comment]
Go after both lol

**Reply by CookieyedCodes** [reply]
Oh I am, don't worry but just trying to get actually implemented 😅

**Comment by meoyawn** [comment, marked as answer]
Listenbox supports chapters now

**Reply by CookieyedCodes** [reply]
> Listenbox supports chapters now

Which chapters do you support?

**Reply by meoyawn** [reply]
> > Listenbox supports chapters now
> 
> Which chapters do you support?

this mechanism: https://youtu.be/YEYnVE-vunI
